_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -2507,6 +2515,11 @@
 /obj/structure/stone_tile/surrounding_tile/cracked,
 /turf/closed/mineral/volcanic/lava_land_surface,
 /area/ruin/unpowered/ash_walkers)
+"WA" = (
+/obj/structure/stone_tile/block/cracked,
+/obj/item/book/granter/crafting_recipe/bone_bow,
```

Comment:
You only ever need one, and your ment to have to trade for one not have to steal it.